WcaOnRails/app/controllers/static_pages_controller.rb

```diff
@@ -5,7 +5,7 @@ class StaticPagesController < ApplicationController
 
   before_action :current_user_is_authorized_for_action!, only: [:panel_wfc]
   private def current_user_is_authorized_for_action!
```

Comment:
This method name tells you nothing about the fact that this authorization is based on WFC.

Reply:
Okay changed.